yoti/YotiHelper.php

```diff
@@ -292,15 +292,26 @@ public static function getFlash()
     /**
      * Generate Yoti unique username.
      *
+     * @param ActivityDetails $activityDetails
      * @param string $prefix
-     * @return string
+     *
+     * @return null|string
      */
-    private function generateUsername($prefix = 'YotiUser-')
+    private function generateUsername(ActivityDetails $activityDetails, $prefix = 'Yoti User')
```

Comment:
Yes that makes sense, but my only concern is if someone calls the function like `generateUsername($activityDetails, 'Test User')`, this default username won't follow the standard of `test.user` instead it will be `Test User`. If that's fine by you I will move it.
I will definitely change `Yoti User` to `yoti.user`